rebench/denoise.py

```diff
@@ -19,39 +19,7 @@
     rebench_version = "unknown"
 
 
-class CommandsPaths:
```

Comment:
Not sure what's going on here, but this looks like you're reverting a change that was merged yesterday.